echopype/convert/set_groups_ad2cp.py

```diff
@@ -254,68 +252,220 @@ def set_platform(self) -> xr.Dataset:
+            beam_groups.append(ds)
+
+            self._beamgroups.append(
+                {
+                    "name": f"Beam_group{len(self._beamgroups) + 1}",
+                    "descr": (
+                        "contains backscatter echo intensity and other configuration "
+                        "parameters from the Echosounder mode. "
+                        "Data can be pulse compressed or raw intensity."
+                    ),
+                }
+            )
+        # echosounder raw
+        if BeamGroup.ECHOSOUNDER_RAW in beam_groups_exist:
+            beam_groups.append(
+                self._make_dataset(
+                    {
+                        "num_beams": "number_of_beams",
+                        "coordinate_system": "coordinate_system",
+                        "num_cells": "number_of_cells",
+                        "blanking": "blanking",
+                        "cell_size": "cell_size",
+                        "velocity_range": "velocity_range",
+                        "echosounder_frequency": "echosounder_frequency",
+                        "ambiguity_velocity": "ambiguity_velocity",
+                        "dataset_description": "data_set_description",
+                        "transmit_energy": "transmit_energy",
+                        "velocity_scaling": "velocity_scaling",
+                        "echosounder_raw_samples_i": "backscatter_r",
+                        "echosounder_raw_samples_q": "backscatter_i",
+                        "echosounder_raw_transmit_samples_i": "transmit_pulse_r",
+                        "echosounder_raw_transmit_samples_q": "transmit_pulse_i",
+                        "echosounder_raw_beam": "echosounder_raw_beam",
+                        "echosounder_raw_echogram": "echosounder_raw_echogram",
+                    }
+                )
+            )
+
+            self._beamgroups.append(
+                {
+                    "name": f"Beam_group{len(self._beamgroups) + 1}",
+                    "descr": (
+                        "contains complex backscatter raw samples and other configuration "
+                        "parameters from the Echosounder mode, "
+                        "including complex data from the transmit pulse."
+                    ),
+                }
+            )
 
         # FIXME: this is a hack because the current file saving
         # mechanism requires that the beam group have ping_time as a dimension,
         # but ping_time might not be a dimension if the dataset is completely
         # empty
-        if "ping_time" not in ds.dims:
-            ds = ds.expand_dims(dim="ping_time")
+        for i, ds in enumerate(beam_groups):
+            if "ping_time" not in ds.dims:
+                beam_groups[i] = ds.expand_dims(dim="ping_time")
 
-        return set_encodings(ds)
+        # remove time1 from beam groups
+        for i, ds in enumerate(beam_groups):
+            beam_groups[i] = ds.sel(time1=ds["ping_time"])
```

Comment:
Drop the `time1` coordinate after this selection.

Right now:
![Screen Shot 2022-06-19 at 7 48 26 PM](https://user-images.githubusercontent.com/15334215/174516651-f431dd08-c038-4045-ab31-60bbcfbcb576.png)